src/main/java/com/hw/bookstore/validation/BookValidation.java

```diff
@@ -0,0 +1,20 @@
+package com.hw.bookstore.validation;
+
+import com.hw.bookstore.domain.repository.BookRepository;
+import com.hw.bookstore.exception.BookNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class BookValidation {
+
+    private final BookRepository bookRepository;
+
+    public void validateBookExistence(Long id) {

```

Comment:
Do you really need a separate class for this method? 
Don't forget validate BookRequestDto using annotations ;)